data_pipeline/pipeline.py

```diff
@@ -1,17 +1,18 @@
 """Parse GPX files when uploaded to an S3 bucket, and insert the data into a SQLite database."""
 
-from contextlib import closing
-from io import BytesIO
 import json
 import os
 import sqlite3
 import xml.etree.ElementTree as xml
+from contextlib import closing
+from io import BytesIO
+from pathlib import Path
 
 import autokitteh
 from autokitteh.aws import boto3_client
 
-
-DB_DSN = os.getenv("DB_DSN")  # Secret
+DB_DSN = os.getenv('DB_DSN')  # Secret
+CREATE_DB = os.getenv('CREATE_DB', 'no').lower() in {'y', 'yes', 'true'}
```

Comment:
Good idea, can you do it? (not working until Sunday)

Reply:
Sure, but I have another question. If user is on cloud and sets `CREATE_DB` = 'yes', then what is the `DB_DSN`? I don't think it's clear in this workflow what the user should do with that env.

I think we either need to add some documentation specifying what file paths are available, or checking that if `CREATE_DB` is set to 'yes' then defaulting to something like this:
```
   code_dir = Path(__file__).absolute().parent
   DB_DSN = code_dir / "data" / "hikes.db"
```

Reply:
Maybe something like:

```python
code_dir = Path(__file__).parent
DB_DSN = os.getenv('DB_DSN') or code_dir / 'hikes.db'  # Secret
```

I don't want the extra `data` directory since we'll need to create it and it's an extra step that's not relevant to what we're trying to show.